csv_download.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+$client_curl = curl_init();
+
+curl_setopt_array($client_curl, array(
+    CURLOPT_URL => "https://api.cookiebot.com/auth/v1/connect/token",
+    CURLOPT_SSL_VERIFYPEER => false,
+    CURLOPT_RETURNTRANSFER => true,
+    CURLOPT_CUSTOMREQUEST => "POST",
+    CURLOPT_POSTFIELDS => "grant_type=client_credentials&client_id=" . get_option('client_ID') . "&client_secret=" . get_option('client_Secret'),
+    CURLOPT_HTTPHEADER => array(
+        "Content-Type: application/x-www-form-urlencoded"
+    ),
+));
+
+$client_response = curl_exec($client_curl);
+
+curl_close($client_curl);
+
+$token = json_decode($client_response);
+
+
+
+
+
+$CSV_curl = curl_init();
+
+curl_setopt_array($CSV_curl, array(
+CURLOPT_URL => "https://app-cookiebot-api-umbracodashboard-prod.azurewebsites.net/urls/" . get_option('domainID') . "?format=json",
+CURLOPT_RETURNTRANSFER => true,
+CURLOPT_SSL_VERIFYPEER => false,
+CURLOPT_ENCODING => "",
+CURLOPT_MAXREDIRS => 10,
+CURLOPT_TIMEOUT => 0,
+CURLOPT_FOLLOWLOCATION => true,
+CURLOPT_HTTP_VERSION => CURL_HTTP_VERSION_1_1,
+CURLOPT_CUSTOMREQUEST => "GET",
+CURLOPT_HTTPHEADER => array(
+    "Authorization: Bearer " . $token->access_token
+),
+));
+
+$CSV_response = curl_exec($CSV_curl);
+
+curl_close($CSV_curl);
+
+$CSV = json_decode($CSV_response);
+
+$fileName = 'somefile.csv';
```

Comment:
Might want to use a more meaningful filename? Something like `domains-{client_id}-{date}.csv` for example.